gratipay/utils/team_history.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from datetime import datetime
+from decimal import Decimal as D
+
+
+def get_end_of_year_totals(db, team, year):
+
+    received = db.one("""
+        SELECT COALESCE(sum(amount), 0) AS Received
+          FROM payments
+         WHERE team = %(team)s
+           AND extract(year from timestamp) = %(year)s
+           AND amount > 0
+           AND direction='to-team';
+    """, locals() )
+
+    if received is None:
```

Comment:
Since we're using `COALESCE`, I don't think `received` will ever be `None`. Will it?